content/docs/06-integrations/00-longhorn.md

```diff
@@ -22,7 +22,7 @@ Longhorn creates a dedicated storage controller for each block device volume and
 
 # Prerequisites
 
-- Kubernetes cluster is at least version 1.22 and not higher than 1.24.
+- Kubernetes cluster is 1.25 or higher.
```

Comment:
Kubernetes version for longhorn should be v1.22 at a minimum. 
The first version of longhorn v1.3.1 is not supported for k8s versions 1.25 and higher.
So, we're rolling out a new version of longhorn v1.4.0. 

This is an example where external dependencies (in this case Kubernetes) vary for a specific version of the pack and the pre-req / limitations should be documented for every version.